docs/authentication.md

```diff
@@ -2,25 +2,26 @@
 
-By default, Percona Server for MongoDB provides a  authentication mechanism where clients authenticate themselves by providing their user credentials.
+By default, Percona Server for MongoDB provides a SCRAM authentication mechanism where clients authenticate themselves by providing their user credentials.
 In addition, you can integrate Percona Server for MongoDB with a separate service, such as OpenLDAP or Active Directory. This enables users to access the database with the same credentials they use for their emails or workstations.
 
 You can use any of these authentication mechanisms supported in Percona Server for MongoDB:
 
 
 * [SCRAM](#scram)(default)
 * [x.509 certificate authentication](#x509-certificate-authentication)
-* [LDAP authentication with SASL](sasl-auth.md)
-* [Kerberos Authentication](kerberos.md)
+* [LDAP authentication with SASL](#ldap-authentication-with-sasl)
+* [Kerberos Authentication](#kerberos-authentication)
 * [Authentication and authorization with direct binding to LDAP](authorization.md)
+* [AWS IAM authentication](aws-iam.md)
 
 ## SCRAM
 
-SCRAM is the default authentication mechanism. *Percona Server for MongoDB* verifies the credentials against the user’s name, password and the database where the user record is created for a client (authentication database). For how to enable this mechanism, see [Enabling Authentication](enable-auth.md#enable-auth).
+SCRAM is the default authentication mechanism. *Percona Server for MongoDB* verifies the credentials against the user’s name, password and the database where the user record is created for a client (authentication database). For how to enable this mechanism, see [Enable Authentication](enable-auth.md#enable-auth).
 
 ## x.509 certificate authentication
 
-This authentication mechanism enables a client to authenticate in Percona Server for MongoDB by providing an x.509 certificate instead of user credentials. Each certificate contains the `subject` field defined in the  format. In Percona Server for MongoDB, each certificate has a corresponding user record in the `$external` database. When a user connects to the database, Percona Server for MongoDB matches the `subject` value against the usernames defined in the `$external` database.
+This authentication mechanism enables a client to authenticate in Percona Server for MongoDB by providing an x.509 certificate instead of user credentials. Each certificate contains the `subject` field defined in the DN format. In Percona Server for MongoDB, each certificate has a corresponding user record in the `$external` database. When a user connects to the database, Percona Server for MongoDB matches the `subject` value against the usernames defined in the `$external` database.
 
 For production use, we recommend using valid  certificates. For testing purposes, you can generate and use self-signed certificates.
```

Comment:
valid x.509 certificates? (There are two space characters and it seems something is missing between them)